commands/moderation/tempban.js

```diff
@@ -0,0 +1,108 @@
+            console.log(reason)
+            if (reason === "") {
+                return msg.reply("Please provide a reason for temporarily banning this user.");
+            }
+
+            let now = new Date();
+            let date = dateFormat(now, "yyyy-mm-dd HH:MM:ss");
+            const action = "cc!tempban " + args.join(" ")
+            // Inserts row into database
+            var sql = `INSERT INTO infractions (timestamp, user, action, length_of_time, reason, valid, moderator) VALUES
+            ('${date}', '${toTempBan.id}', 'cc!tempban', '${timeLength}', '${reason}', true, '${msg.author.id}');
+            INSERT INTO mod_log (timestamp, moderator, action, length_of_time, reason) VALUES
+            ('${date}', '${msg.author.id}', '${action}', '${timeLength}', '${reason}');`;
+            con.query(sql, function (err, result) {
+                if (err) {
+                    console.log(err);
+                } else {
+                    console.log("1 record inserted");
+                }
+            });
+            
+            // Sends Audit Log Embed
+            let channel = msg.guild.channels.cache.find(channel => channel.name === 'audit-logs')
+
+            const tempBanEmbed = new Discord.MessageEmbed()
+            .setColor('#0099ff')
+            .setTitle(`${toTempBan.user.username}#${toTempBan.user.discriminator} was banned by ${msg.author.tag} for ${timeLength}:`)
+            .setDescription(reason)
+            .setThumbnail(`https://cdn.discordapp.com/avatars/${toTempBan.user.id}/${toTempBan.user.avatar}.png`)
+            .setTimestamp()
+            .setFooter(`${msg.guild.name}`);
+
+            channel.send(tempBanEmbed)
+
+            const tempUnBanEmbed = new Discord.MessageEmbed()
+            .setColor('#0099ff')
+            .setTitle(`${toTempBan.user.username}#${toTempBan.user.discriminator} was unbanned by after ${timeLength}:`)
+            .setThumbnail(`https://cdn.discordapp.com/avatars/${toTempBan.user.id}/${toTempBan.user.avatar}.png`)
+            .setTimestamp()
+            .setFooter(`${msg.guild.name}`);
+
+            // Banning member and sending him a DM with a form to refute the ban and the reason
+            toTempBan.send("You've been banned for " + timeLength + " for the following reason: ```" + reason + " ``` If you wish to challenge this ban, please submit a response in this Google Form: https://docs.google.com/forms/d/e/1FAIpQLSc1sx6iE3TYgq_c4sALd0YTkL0IPcnkBXtR20swahPbREZpTA/viewform")
+            toTempBan.ban({ reason })
+            
+            msg.reply(`${toTempBan} was banned for ${timeLength}.`)
+
+            setTimeout(() => {
+                msg.guild.members.unban(toTempBan)
+                channel.send(tempUnBanEmbed);
+
+<<<<<<< HEAD
+                toTempBan.send(`You have been unbanned from the Codecademy Community after ${timeLength}`);
+
+=======
+>>>>>>> 57922792b7d64410c43f5df1b07363ee20be2651
```

Comment:
What is `HEAD` and the long code?

Reply:
My bad, sorry, looks like I forgot to commit my changes after fixing that. That was when I rebased and I had some merge conflicts